js/src/views/Signer/components/TransactionPendingForm/TransactionPendingFormConfirm/transactionPendingFormConfirm.js

```diff
@@ -27,7 +27,7 @@ import styles from './transactionPendingFormConfirm.css';
 
 export default class TransactionPendingFormConfirm extends Component {
   static propTypes = {
-    account: PropTypes.object.isRequired,
+    account: PropTypes.object,
```

Comment:
The funny thing here was that (a) the `isRequired` was not true & (b) the tests actually covered up the failure. But yes, making it default empty is probably a better approach than checking each access. 

Will update, thanks.